### ISSUE TITLE
Update Louvain docs

### ISSUE THREAD

**Issue by tomasonjo** [neo4j-contrib/neo4j-graph-algorithms#736]
With the level 2 louvain we now store community of node at each iteration and hence the result is no more an integer, but an array of integers. Also param community has been changed to communities for streaming version

**Comment by jexp** [comment]
Yes I fully agree with Tomaz, the single value param should stay (also for indexing)
And the array (which is more expensive) should be an optional, additional property.

**Comment by tomasonjo** [comment]
Will the last two community ids in the communities array always be the same as the last pass always determines that the algo converged as the values haven't changed and stops iterations.

**Comment by jexp** [comment]
@mknblch we MUST NOT change signatures of procedures (parameters or result columns) that are not backwards compatible.

So keep "community" for both the write and stream version and make the new communities an optional result via config (can default to true, but people need to be aware that much more data is written and streamed back).

See: https://community.neo4j.com/t/graph-algorithms/2333

**Comment by mneedham** [comment]
@jexp  I think I made that change rather than @mknblch. We can make the change as you suggested, but the meaning of 'community' will be a bit different than it was before we implemented 2 stage Louvain.

In the 1 stage Louvain 'community' was referring to the communities found at stage 1 of the algorithm, whereas if we make it use the value from the last iteration of the algorithm that will be different, assuming that it runs for > 1 stage. 

Is it still ok to store the last stage as 'community' given that?
